{"name": "test_add_group[None:nameHtni;headerz;footerVZqSohH]", "status": "broken", "statusDetails": {"message": "AttributeError: module pytest has no attribute allure", "trace": "app = <fixture.application.Application object at 0x00000234A91672B0>\ndb = <fixture.db.DbFixture object at 0x00000234A9166650>, json_groups = None:nameHtni;headerz;footerVZqSohH\ncheck_ui = False\n\n    def test_add_group(app, db, json_groups, check_ui):\n        #group = data_groups\n        group = json_groups\n>       with pytest.allure.step('Given a group list'):\n\ntest\\test_add_group.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nname = 'allure'\n\n    def __getattr__(name: str) -> object:\n        if name == \"Instance\":\n            # The import emits a deprecation warning.\n            from _pytest.python import Instance\n    \n            return Instance\n>       raise AttributeError(f\"module {__name__} has no attribute {name}\")\nE       AttributeError: module pytest has no attribute allure\n\nenv\\lib\\site-packages\\pytest\\__init__.py:165: AttributeError"}, "parameters": [{"name": "json_groups", "value": "None:nameHtni;headerz;footerVZqSohH"}], "start": 1655746690553, "stop": 1655746690554, "uuid": "9ddfdfd7-9792-451b-9844-d4b205535f72", "historyId": "bb4776718b5a65d06b630630763e361f", "testCaseId": "8e9b1d07ef6cdb67de3c4a3d82e10652", "fullName": "test.test_add_group#test_add_group", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_add_group"}, {"name": "host", "value": "DESKTOP-GTBJKMJ"}, {"name": "thread", "value": "12556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_add_group"}]}